rero/rero-ils
# collection: create resource

* Creates collections resource in order to group linked items.
* Creates public detailed view.
* Sorts entry points alphabetically.
* Moves 'Help' submenu in 'Tools' menu

Co-Authored-by: Alicia Zangger <alicia.zangger@rero.ch>
Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>
Co-Authored-by: Renaud Michotte <renaud.michotte@gmail.com>

## Why are you opening this PR?
Task 1726 of US1591
https://tree.taiga.io/project/rero21-reroils/us/1591?milestone=273464

## Dependencies

My PR depends on the following `rero-ils-ui`'s PR(s):

* https://github.com/rero/rero-ils-ui/pull/376

## How to test?

Use https://github.com/rero/rero-ils-ui/pull/376 to have collection brief views.
As an anonymous user, go to an org view and check Tools > Collections (not available for `global` viewcode)
Same with a patron and a librarian

## Code review check list
